routemaster/application.rb

```diff
@@ -18,7 +17,11 @@ class Application < Sinatra::Base
       set :raise_errors, false
     end
 
-    use Rack::SSL
+    if ENV.fetch('FORCE_SSL', 'true') =~ /YES|TRUE|ON|1/i
+      require 'rack/ssl'
```

Comment:
Ok, but if you want to lazily require it, then you can also add `require: false` in the gemfile.